.github/workflows/build_lint.yml

```diff
@@ -5,7 +5,13 @@ on:
     branches:
     - main
     - release**
+    paths-ignore:
```

Comment:
Just wanting to understand this behavior:
- If the PR contains a mixture of .md and .py changes, this job will run
- If only .md, this job won't run
- If only .py, this job will run
Is this correct?

Reply:
OK, the doc spelled it out clearly:
> When all the path names match patterns in paths-ignore, the workflow will not run. If any path names do not match patterns in paths-ignore, even if some path names match the patterns, the workflow will run.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpaths